### ISSUE TITLE
Add option to automatically create new release

### ISSUE THREAD

**Issue by KPostOffice** [thoth-station/kebechet#1062]
## Problem statement
With the new feature which creates tags after a release PR is merged, some users may also want to automatically generate a release as well.

## High-level Goals
Give users a configuration option to automatically create a release when the new version tag is pushed.

## Proposal description
Create new release using OGR/GitHub api **right after** a version tag is created. It should just contain change logs.

## Acceptance Criteria

- [ ] Add concrete acceptance criteria

**Comment by VannTen** [comment]
/lifecycle frozen
/remove-triage accepted
(Since the acceptance critera are not yet concrete)

[The docs](https://thoth-station.ninja/docs/developers/kebechet/managers/version.html) says that kebechet does not create release, so maybe that would mean creating a "releases managers", separate from the version manager ?

**Comment by harshad16** [comment]
yeah corect, either we would extend version to create the github release or have a separate release manager, I think it would be left on convenience and easy of fitting in the structure of kebechet
